app/src/main/java/org/jellyfin/androidtv/ui/shared/StrokeTextView.kt

```diff
@@ -0,0 +1,67 @@
+package org.jellyfin.androidtv.ui.shared
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.core.content.ContextCompat
+import org.jellyfin.androidtv.R
+
+class StrokeTextView : AppCompatTextView {
```

Comment:
Constructor should be the same as the "ServerButtonView" class (including the overloads annotation).